### PR TITLE
redirect /community to pubpub

### PR DESCRIPTION
**Pull request by ali-tm-amin**

### CODE REVIEW

File: salt/journal/config/etc-nginx-sites-available-journal.conf [hunk 1]
```diff
@@ -291,7 +291,46 @@ server {
...
+    # "/community/ambassadors"                     => proxy (as /community/ambassadors)
+    # "/community/community-alumni"                => proxy (as /community/community-alumni)
+    # "/community/community-voices"                => proxy (as /community/community-voices)
+    
+
+    location ^~ /community {
+        # rewrite /community to /community/
+        rewrite '^/community$' '{{ public_scheme }}://{{ public_host }}/community/' permanent;
+        # Rewrite a page that existed before, but has now been merged with the pubpub pages
+        rewrite '^/community/publishing-with-elife$' '{{ public_scheme }}://{{ public_host }}/community/peer-review' permanent;
```
Comment (scottaubrey) [inline salt/journal/config/etc-nginx-sites-available-journal.conf:314]: This looks unnecessary, maybe copied with the about page stuff?

File: salt/journal/config/etc-nginx-sites-available-journal.conf [hunk 2]
```diff
@@ -291,7 +291,46 @@ server {
...
     {% endif -%}
 
-    {% if pillar.journal.about_url or pillar.journal.resources_url or pillar.journal.media_policy_url %}
+    {% if pillar.journal.community_url %}
+    # Proxy /community pages to pubpub
+    # - https://github.com/elifesciences/issues/issues/8516
+    # ali-tm-amin@2024-04:  Proxy all community pages below to pubpub
+    # "/community"                                 => proxy (as /community)
+    # "/community/"                                => proxy (as /community/)
+    # "/community/get-involved"                    => proxy (as /community/get-involved)
```
Comment (scottaubrey) [inline salt/journal/config/etc-nginx-sites-available-journal.conf:300]: each of these examples has the wrong proxy path for the right hand side

**Review by scottaubrey — CHANGES_REQUESTED** [review]
Some small docs and maybe a line that needs removed, but looks alright to me. If you agree with the suggestions, feel free to make them and merge when you're ready.

File: salt/journal/config/etc-nginx-sites-available-journal.conf (hunk 2 above)
Comment (ali-tm-amin) [reply salt/journal/config/etc-nginx-sites-available-journal.conf]: Corrected, thanks

**Pull request merged**